hmt_escrow/storage.py

```diff
@@ -29,6 +29,9 @@
 ESCROW_AWS_ACCESS_KEY_ID = os.getenv("ESCROW_AWS_ACCESS_KEY_ID", "minio")
 ESCROW_AWS_SECRET_ACCESS_KEY = os.getenv("ESCROW_AWS_SECRET_ACCESS_KEY", "minio123")
 
+ESCROW_AWS_S3_ACCESS_KEY_ID = os.getenv("ESCROW_AWS_S3_ACCESS_KEY_ID", "")
```

Comment:
No, it's a user with programmatic access to S3 with read and write permissions, it will be used for write/read to/from public and private (in future) buckets.

Reply:
May be `ESCROW_RESULTS_AWS_S3_ACCESS_KEY_ID` ?

Reply:
ok

Reply:
you are right. you don't have to change anything here. ty

Reply:
Changed to `ESCROW_RESULTS_AWS_S3_...` for more transparency